### ISSUE TITLE
Maybe chapter list display mode could also be used in transition screen?

### ISSUE THREAD

**Issue by curche** [tachiyomiorg/tachiyomi#9385]
### Steps to reproduce

1. Find a random source (MangaDex)
2. Find a random manga not in library (Aim for the Ace!)
3. Click on Filter (or the more secret way: tap on the Total Chapter Count: 20 chapters currently)
4. Go to Display: Make sure its changed to "Chapter number"
5. Observe how the chapter list now shows names (Chapter 20, Chapter 19, Chapter 18...)
6. Open a random chapter (Chapter 6)
7. Notice the top bar of the reader shows the chapter name as Vol 1 Ch 6
8. Go to transition screen by just going backwards

### Expected behavior

The transition screen also shud just show
        Previous: Chapter 5
        Current: Chapter 6

### Actual behavior

The transition screen shows up like this
        Previous: Vol 1 Ch 5 | Horohoro Tamago
        Current: Vol 1 Ch 6 | Horohoro Tamago

### Crash logs

N/A

### Tachiyomi version

r5707

### Android version

Android 13

### Device

SM-N770F

### Other details

While the bug is reported for transition screen, it could also apply to the top bar of the reader as well tbh

This was reported in Discord the other day and I felt this was a reasonable request
Haven't found similar issues other than https://github.com/tachiyomiorg/tachiyomi/issues/7414 (maybe I might have missed something)
Tried building the app from source to take a dig at it but Android Studio/Gradle refuses to accept the new hierarchy somehow
Might as well report it in case anyone else is able to fix this. Good Luck!

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/help/faq/) and [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by staxhinho** [comment]
I don't see anything wrong, but if i missed something can you explain better?

**Comment by stevenyomi** [comment]
tl;dr: Show simple "Chapter X" instead of full chapter name in transition screen if you set this in chapter list.
